daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -2067,3 +2067,44 @@ def _resume_device(self):
         _da_set_logo_brightness(self, logo_brightness)
         _da_set_scroll_brightness(self, scroll_brightness)
         self.disable_notify = False
+
+
+class RazerAtheris(__RazerDevice):
+    """
+    Class for the Razer Atheris
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0062
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Razer_Atheris_-_Mobile_Gaming_Mouse-if0(1|2)-event-kbd')
+    METHODS = ['get_device_type_mouse', 'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_poll_rate', 'set_poll_rate',
+               # Battery
+               'get_battery', 'set_idle_time', 'set_low_battery_threshold']
```

Comment:
no `is_charging`?

Reply:
It uses 2xAA batteries.

Reply:
please add is_charging like done in the Basilisk X Hyperspeed (incl. hardcoded "0" in driver)

Reply:
Ok, it is added :)